### PR TITLE
time ranges

### PR DESCRIPTION
**Pull request by romanmandryk**

### CODE REVIEW

File: server/Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ openssl                 = "0.10"
 ordered-float           = "3.2"
 prost                   = "0.11"
 prost-types             = "0.11"
-router                  = { git = "https://github.com/Arrow-air/lib-router.git", branch = "main" }
+router                  = { git = "https://github.com/Arrow-air/lib-router.git", branch = "roman/r2_time_ranges" }
```
Comment (ServiceDog) [inline server/Cargo.toml:27]: this can now be updated to develop branch

**Review by ServiceDog — CHANGES_REQUESTED** [review]
Thanks! I like `earliest_depart` and `latest_arrival`, makes sense from the customer perspective. Mine had min/max for departure and min/max for arrival, I think this implementation is cleaner.

**Comment by owlot** [comment]
This branch needs a rebase from develop: `git pull --rebase origin develop`

File: server/Cargo.toml (hunk 1 above)
Comment (romanmandryk) [reply server/Cargo.toml]: I updated it

**Review by ServiceDog — APPROVED** [review]

**Pull request merged**